backend/.gitignore

```diff
@@ -9,4 +9,5 @@ yarn.lock
 pnpm-lock.yaml
 bun.lockb
 .env
-wrangler.toml
\ No newline at end of file
+wrangler.toml
+wrangler.sample.toml
```

Comment:
No need of adding this in .gitignore

Reply:
wrangler.toml was by default added in the original repo I removed and added it by mistake git detected it as a addition. wrangler.sample.toml was added as a extra I have removed it. Also removed the commented out url.